rbi/core/marshal.rbi

```diff
@@ -205,6 +205,13 @@ module Marshal
   #
   # Never pass untrusted data (including user supplied input) to this method.
   # Please see the overview for further details.
+  sig do
+    params(
+        arg0: IO,
+        arg1: Proc,
+    )
+    .returns(Object)
+  end
```

Comment:
This should not be an overload, it should be a `T.any` for the type of the argument.

Reply:
That makes sense. Fixed!